Source/Scene/Camera.js

```
@@ -2451,37 +2451,96 @@ define([
+     * The offset is heading/pitch/range in the local east-north-up reference frame centered at the center of the bounding sphere.
+     * The heading and the pitch angles are defined in the local east-north-up reference frame.
+     * The heading is the angle from y axis and increasing towards the x axis. Pitch is the rotation from the xy-plane. Positive pitch
+     * angles are above the plane. Negative pitch angles are below the plane. The range is the distance from the center. If the range is
+     * zero, a range will be computed such that the whole bounding sphere is visible.
+     *
+     * In 2D, there must be a top down view. The camera will be placed above the target looking down. The height above the
+     * target will be the range. The heading will be determined from the offset. If the heading cannot be
+     * determined from the offset, the heading will be north.
      *
      * @param {BoundingSphere} boundingSphere The bounding sphere to view.
-     * @param {Object} [options] Object with the following properties:
+     * @param {HeadingPitchRange} [offset] The offset from the target in the local east-north-up reference frame centered at the target.
      */
-    Camera.prototype.viewBoundingSphere = function(boundingSphere, options) {
+    Camera.prototype.viewBoundingSphere = function(boundingSphere, offset) {
         //>>includeStart('debug', pragmas.debug);
         if (!defined(boundingSphere)) {
             throw new DeveloperError('boundingSphere is required.');
         }
         //>>includeEnd('debug');
 
-        var transform = Transforms.eastNorthUpToFixedFrame(boundingSphere.center);
-        this.transform = transform;
-        var r = 2.0 * Math.max(boundingSphere.radius, this.frustum.near);
-        this.lookAt(new Cartesian3(0, -r, r), Cartesian3.ZERO, Cartesian3.UNIT_Z);
-        this.setTransform(Matrix4.IDENTITY);
+        if (this._mode === SceneMode.MORPHING) {
+            return;
```

This should probably throw a `DeveloperError` instead of simply returning.